randomwallpaper@iflow.space/settings.ui

```diff
@@ -645,6 +645,103 @@ You can also define a prefix that will be added to the image URL.</property>
                   </packing>
                 </child>
+                <child>
+                  <object class="GtkFrame">
+                    <property name="visible">True</property>
+                    <property name="can-focus">False</property>
+                    <property name="label-xalign">0</property>
+                    <property name="shadow-type">in</property>
+                    <child>
+                      <!-- n-columns=2 n-rows=1 -->
+                      <object class="GtkGrid">
+                        <property name="visible">True</property>
+                        <property name="can-focus">False</property>
+                        <property name="margin-start">15</property>
+                        <property name="margin-end">15</property>
+                        <property name="margin-top">15</property>
+                        <property name="margin-bottom">15</property>
+                        <property name="column-spacing">10</property>
+                        <property name="baseline-row">2</property>
+                        <child>
+                          <!-- n-columns=1 n-rows=2 -->
+                          <object class="GtkGrid">
+                            <property name="visible">True</property>
+                            <property name="can-focus">False</property>
+                            <property name="hexpand">True</property>
+                            <property name="column-homogeneous">True</property>
+                            <child>
+                              <object class="GtkLabel" id="enable-image-stretching-label">
+                                <property name="visible">True</property>
+                                <property name="can-focus">False</property>
+                                <property name="halign">start</property>
+                                <property name="label" translatable="yes">Enable image stretching</property>
+                              </object>
+                              <packing>
+                                <property name="left-attach">0</property>
+                                <property name="top-attach">0</property>
+                              </packing>
+                            </child>
+                            <child>
+                              <object class="GtkLabel" id="enable-image-stretching-description">
+                                <property name="visible">True</property>
+                                <property name="can-focus">False</property>
+                                <property name="hexpand">True</property>
+                                <property name="label" translatable="yes">This option enables the image stretching. It is useful when you have dual screens, you could set the allowed resolution with doubled width (for example 3840×1080 in case of two Full-HD monitors) and enable stretching. In that case, a single image will be displayed on both screens.</property>
```

Comment:
```suggestion
                                <property name="label" translatable="yes">This option enables image stretching. It is useful for stretching the image over multiple screens. Set the resolution to the sum of your monitors (for example 3840×1080 in case of two Full-HD monitors) and enable stretching. In that case, a single image will be displayed on both screens.</property>
```

Small writing suggestions :)